docs/reference/ml/apis/delete-expired-data.asciidoc

```diff
@@ -0,0 +1,47 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ml-delete-expired-data]]
+=== Delete Expired Data API
+++++
+<titleabbrev>Delete Expired Data</titleabbrev>
+++++
+
+Deletes expired and unused machine learning data.
+
+==== Request
+
+`DELETE _xpack/ml/_delete_expired_data`
+
+==== Description
+
+Deletes all job results, model snapshots and forecast data that have exceeded their
+`retention days` period.
+Machine Learning state documents that are not associated with any job are also deleted.
+
+==== Authorization
+
+You must have `manage_ml`, or `manage` cluster privileges to use this API.
+For more information, see
+{xpack-ov}/security-privileges.html[Security Privileges] and
```

Comment:
Should `xpack-ov` be `stack-ov` in this line?

Reply:
Yes, corrected